dvc/logger.py

```diff
@@ -62,7 +63,7 @@ def format(self, record):
             return record.msg
 
         if record.levelname == "ERROR" or record.levelname == "CRITICAL":
-            exception, stack_trace = self._parse_exc(record.exc_info)
+            exception, stack_trace = self._parse_exc(record)
```

Comment:
Also looked into standard `formatException` again (since py2 was dropped), but it doesn't quite do what we need. :slightly_frowning_face:

Reply:
yeah :disappointed:  I was working on a PR to refactor that but couldn't make it work as intended.